openpower/package/sb-signing-utils/sb-signing-utils.mk

```diff
@@ -4,11 +4,14 @@
 #
 ################################################################################
 
+
 SB_SIGNING_UTILS_SITE ?= $(call github,open-power,sb-signing-utils,$(SB_SIGNING_UTILS_VERSION))
 
 SB_SIGNING_UTILS_LICENSE = Apache-2.0
 SB_SIGNING_UTILS_LICENSE_FILES = LICENSE
-SB_SIGNING_UTILS_VERSION ?= v0.8
+
+# MAB can either use the direct commit or v0.9 if this commit gets marked with that
```

Comment:
Remove this commit or change to not include MAB :)

Reply:
This comment line will be removed once I hear back from @cjengel about where to directly link to the commit number, or if there will be a new "v0.9" release.

Reply:
v0.9 is available (page didn't refresh cleanly for me before) so I will be using that.

Reply:
fixed in latest patchset